lib/indexer/tag_lookup.rb

```diff
@@ -30,12 +30,16 @@ def find_content_id(doc_hash)
       if doc_hash["content_id"].present?
         doc_hash["content_id"]
       else
-        Services.publishing_api.lookup_content_id(base_path: doc_hash["link"])
+        GdsApi.with_retries do
```

Comment:
Can we explicitly specify the number of retries here? Otherwise you can't see what it will do by just glancing at the code.